vars/unitTest.groovy

```diff
@@ -103,9 +103,9 @@ Map afterTest(Map config, Map testRunInfo) {
         fileOperations([fileCopyOperation(excludes: '',
                                       flattenFiles: false,
                                       includes: valgrind_pattern,
-                                      targetLocation: target_dir)])
+                                      targetLocation: memcheck_dir)])
         sh label: 'Create tarball of Valgrind xml files',
-           script: "tar -czf ${target_dir}.tar.gz ${target_dir}"
+           script: "tar -czf ${memcheck_dir}.tar.gz ${memcheck_dir}"
```

Comment:
Minor but bzip would be better here than gz as it'll take up less space and we archive these files for a while.